webapp/src/plugin.jsx

```diff
@@ -127,7 +127,7 @@ export default class DemoPlugin {
             },
         );
 
-        registry.registerAdminConsoleCustomSetting('SecretMessage', SecretMessageSetting, 'Secret Message');
+        registry.registerAdminConsoleCustomSetting('SecretMessage', SecretMessageSetting, true);
```

Comment:
Should we accept an options `param` instead?

Reply:
What options did you have in mind?

Reply:
I've wrapped the showTitle in an options object and is passed as an anonymous  object where it clearly indicates the `showTitle` is true.